pkgs/os-specific/linux/open-isns/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, openssl, fetchurl }:
+stdenv.mkDerivation rec {
+  name = "open-isns-${version}";
+  version = "0.95";
+
+  src = fetchurl {
```

Comment:
Please use `fetchFromGitHub` or at least `fetchzip`; github tends to change their generated tarballs in various ways (compression-level, for example), resulting in unstable checksums.